Jenkinsfile

```diff
@@ -123,6 +126,37 @@ pipeline{
 								sh "scripts/dev_cli.sh tests --integration --libc musl"
 							}
 						}
+						stage ('Install azure-cli') {
+							steps {
+								installAzureCli("bionic", "arm64")
+							}
+						}
+						stage ('Download Windows image') {
+							steps {
+								sh '''#!/bin/bash -x
+									IMG_PATH=$HOME/workloads/windows-11-iot-enterprise-aarch64.raw
+									IMG_SHA1_SUM_FL="$IMG_PATH.sha1"
+									if test -f "$IMG_SHA1_SUM_FL" && sha1sum -c "$IMG_SHA1_SUM_FL"; then
+										exit
+									fi
+									mkdir -p ${env.HOME}/workloads
+									az storage blob download \
+										--container-name private-images \
+										--file "$IMG_PATH" \
+										--name windows-11-iot-enterprise-aarch64.raw \
+										--connection-string "$AZURE_CONNECTION_STRING"
+									sha1sum "$IMG_PATH" > "$IMG_SHA1_SUM_FL"
+								'''
+							}
+						}
+						stage ('Run Windows guest integration tests') {
+							options {
+								timeout(time: 1, unit: 'HOURS')
```

Comment:
May want to bump this to 2 or 1.5 hours?

Reply:
Or you could not try and download it on the CI. You will block the CI for everyone. Instead just assume that the image is already on the machine.

Reply:
Yep, this should probably be a good step as long as the image is too big.

Thanks

Reply:
Done.

Reply:
I pointed to the wrong stage. The value you bumped is for the actual testing. The downloading stage did not specify any timeout value so it was just using the default.

I'm not sure how long the default value is. If it is only a few minutes then it perhaps can be bump.

If it is already one hour then uploading the image before hand is perhaps better. If you go this route, please keep the hash validation part so that we know if the image is what we expect.